dataprofiler/profilers/profile_builder.py

```diff
@@ -1374,10 +1391,10 @@ def _update_profile_from_chunk(self, data, sample_size,
         elif isinstance(data, list):
             data = pd.DataFrame(data)
 
-        if len(data.columns) != len(data.columns.unique()):
-            raise ValueError('`StructuredProfiler` does not currently support '
-                             'data which contains columns with duplicate '
-                             'names.')
+        duplicate_cols_given = len(data.columns) != len(data.columns.unique())
+        num_cols = [len(idxs) for idxs in self._col_name_to_idx.values()]
+        duplicate_cols_present = any([num > 1 for num in num_cols])
```

Comment:
same as above, should we just have this stored as known info private to the profiler?